### PR TITLE
Cards for TT_SStt search

### PR DESCRIPTION
**Pull request by dpilipov**
Dear MadGraph MC experts, my appologies if I am not doing the pull request correctly - please let me know if I need to do it differently somehow!  

We are currently generating local samples for pair production of vector like tops, each decaying to a new scalar and a standard model top, with a consequent all-hadronic top decay and a diphoton decay of one scalar with the second inclusive.  This is a 2d mass bump search, requiring a 2d mass grid for MC generation (VL top masses are found in masspoints_T.txt and scalar masses are found in masspoints_S.txt). 

We are using a new pair production VLQ model for CMS, eVLQ_v8.1.tar.gz, found at https://feynrules.irmp.ucl.ac.be/wiki/NLOModels, as you can see from TT_SStt_extramodels.dat .  What do we need to do with regards to the use of this new model?  Our fragment.py currently accesses this model locally.

### CODE REVIEW

**Comment by dpilipov** [comment]
Here is the link to our repository for generating the samples given the gridpack: https://github.com/mroguljic/generateCMS/tree/TPrimePair/one_step.  And here is the link to the fragment we are using: https://github.com/mroguljic/generateCMS/blob/TPrimePair/one_step/inputs/B2G-RunIISummer20UL16wmLHEGEN-02702-fragment.py.

**Comment by menglu21** [comment]
Hi @dpilipov I uploaded the model to our central repository and it can be used directly now. Then you need to change "VLQS012_v8_4FNS_UFO.tar" to "eVLQ_v8.1.tar.gz" in the extramodel file.

File: TT_SStt_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+define bx = b b~
+define lx = e- mu- e+ mu+ ve vm ve~ vm~
+define qx = u c d s u~ c~ d~ s~ 
+define phix = s0 z h w+ w-
+define tbx = t t~ b b~
+define wx = w+ w-
+generate p p > tp tp~, ( tp > s0 t, ( s0 > a a ) , ( t > qx qx bx ) ), ( tp~ > s0 t~, ( t~ > qx qx bx ) )
+#DP FOR GENERATING T' CROSS SECTIONS
+#generate p p > tp tp~
+output TT_SStt -nojpeg
```
Comment (menglu21) [inline TT_SStt_proc_card.dat:17]: please remove the commented lines. By using the decay chain syntax, what's the size of the gridpack and how many files after untaring the gridpack

**Review by menglu21 — COMMENTED** [review]
please move all the files to bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV, and modify the run_card referring to other files under that directory.

File: TT_SStt_proc_card.dat (hunk 1 above)
Comment (dpilipov) [reply TT_SStt_proc_card.dat:12]: Size of gridpack: 22917232, number of files after untaring the gridpack: 2074

**Comment by dpilipov** [comment]
Hello MG experts.  I edited the proc and model cards and moved all the cards to bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV as per your request.  Is there anything else i still need to do?

File: TT_SStt_proc_card.dat (hunk 1 above)
Comment (menglu21) [reply TT_SStt_proc_card.dat]: so the size is around 23M?

Comment (dpilipov) [reply TT_SStt_proc_card.dat]: Hi! To answer your question re size - Yes. I do have a question for you! In the customizecards.dat I included the masses which are to be varied across the 2d grid, but I did not include the widths and coupling values corresponding to our param_card.dat.   Should I have included these?  Please let me know if I need to update the customizecards.dat!  Thank you for all your help!

Comment (dpilipov) [reply TT_SStt_proc_card.dat]: Hello experts!  I would like to add my param_card.dat, but I am not sure where I should put the card?  I no longer see any of my cards?!?  I wonder if you might be able to give me a status for TT_SStt cards?  Thanks so much!

**Comment by menglu21** [comment]
Hi @dpilipov please create a directory e.g. "TT_SStt" to store those cards

**Comment by dpilipov** [comment]
Hello experts - I created the directory with the cards - had to do it as a new pull request.  Please see https://github.com/cms-sw/genproductions/pull/3643 .  Thank you!

**Pull request closed without merging**